bundles/org.openhab.binding.nobohub/src/main/resources/OH-INF/i18n/nobohub_no.properties

```
@@ -0,0 +1,71 @@
+thing-type.config.nobohub.nobohub.hostName.description = Tjener eller IP addresse til Nob� Hub
+thing-type.config.nobohub.nobohub.keepaliveInterval.label = Tidsintervall
+thing-type.config.nobohub.nobohub.keepaliveInterval.description = Tidsintervall (sekunder). Standardinstilling: 14
+thing-type.config.nobohub.zone.id.label = Id
+thing-type.config.nobohub.zone.id.description = Id for sone
+thing-type.config.nobohub.component.serialNumber.label = Serialnummer
+thing-type.config.nobohub.component.serialNumber.description = Serialnummer for komponent (12 tall)
+
+
+# channel types
+
+channel-type.nobohub.activeOverrideName-channel-type.label = Aktiv Overstyring
+channel-type.nobohub.activeOverrideName-channel-type.description = Modus p� aktiv overstyring
+channel-type.nobohub.activeOverrideName-channel-type.state.option.NORMAL = Normal
+channel-type.nobohub.activeOverrideName-channel-type.state.option.COMFORT = Komfort
+channel-type.nobohub.activeOverrideName-channel-type.state.option.ECO = Eco
+channel-type.nobohub.activeOverrideName-channel-type.state.option.Away = Borte
+channel-type.nobohub.eco-temperature-channel-type.label = Eco Temperatur
+channel-type.nobohub.eco-temperature-channel-type.description = �nsket Eco temperatur niv� satt p� panel eller i binding
+channel-type.nobohub.comfort-temperature-channel-type.label = Komfort Temperatur
+channel-type.nobohub.comfort-temperature-channel-type.description = �nsket Komfort temperatur niv� satt p� panel eller i binding
+channel-type.nobohub.temperature-channel-type.label = N�v�rende Temperatur
+channel-type.nobohub.temperature-channel-type.description = N�v�rende temperatur fra en enhet som st�tter rapportering
+channel-type.nobohub.activeWeekProfileName-channel-type.label = Aktiv Ukeprofil Navn
+channel-type.nobohub.activeWeekProfileName-channel-type.description = Navn p� n�v�rende aktiv ukesprofil
+channel-type.nobohub.NactiveWeekProfile-channel-type.label = Aktiv Ukeprofil Id
+channel-type.nobohub.weekProfiles-channel-type.label = Ukeprofiler
+channel-type.nobohub.weekProfiles-channel-type.description = Tilgjengelige ukesprofiler, satt opp via Nob� app
+channel-type.nobohub.activeWeekProfile-channel-type.label = Aktiv Ukeprofil Id
+channel-type.nobohub.activeWeekProfile-channel-type.description = Id p� n�v�rende aktiv ukesprofil
+
+
+# User Messages
+message.missing.serial = Mangler serialnummer i konfigurasjon
+message.bridge.status.failed = Kunne ikke hente status: {0}
+message.bridge.missing.hostname = Mangler tjenernavn i konfigurasjon
+message.bridge.connection.failed = Kunne ikke koble til, sjekk nettverksforbindelsen og konfigurasjon
+message.component.illegal.serial = Serialnummer er ukjent eller feil: {0}
+message.component.notfound = Kunne ikke finne Komponent med serialnummer {0} for kanal {1}
+message.component.missing.id = Id er ikke satt for kanal {0}
+message.zone.notfound = KUnne ikke finne Sone med id {0} for kanal {1}
```

```suggestion
message.zone.notfound = Kunne ikke finne Sone med id {0} for kanal {1}
```

Side note: Normally translations should be provided through Crowdin - see https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations. As this is a new binding we can make an exception, since there can be no collisions. It does mean though that the proof-reading process will be short-circuited as this is performed in Crowdin, not through pull request reviews. I don't know if there currently are any active Norwegian translators.

---

Nice to know. Fixed the typo.